src/language-css/parser-postcss.js

```diff
@@ -298,6 +298,23 @@ function parseNestedCSS(node, options) {
       node.raws.value = selector;
     }
 
+    // Move comments that are stuck in between into value for sass only
+    if (
+      node.raws.value &&
+      node.raws.value.scss &&
+      node.raws.between &&
+      node.raws.between.match(/\/\/|\/\*/)
```

Comment:
`hasInlineComment`

Reply:
Hmm I can't seem to find that function in the codebase?

Reply:
Yes, please move it to utils and rename the function